schemas/input/csl-data.json

```diff
@@ -310,6 +310,9 @@
       "event": {
         "type": "string"
```

Comment:
Can we use here a `description` field to indicated that this is deprecated?

Reply:
We should be adding `title` and `description` properties as we go, per #190.

Reply:
How do we want to keep the schema description and rst documentation in sync? Or do we leave the schema description as a shorter brief description?